Corona-Warn-App/src/main/java/de/rki/coronawarnapp/submission/ExposureKeyHistoryCalculations.kt

```diff
@@ -1,39 +1,72 @@
 package de.rki.coronawarnapp.submission
 
+import KeyExportFormat
+import androidx.annotation.VisibleForTesting
 import com.google.android.gms.nearby.exposurenotification.TemporaryExposureKey
+import org.joda.time.Days
+import org.joda.time.Instant
 
 class ExposureKeyHistoryCalculations(
     private val transmissionRiskVectorDeterminator: TransmissionRiskVectorDeterminator,
+    private val daysSinceOnsetOfSymptomsVectorDeterminator: DaysSinceOnsetOfSymptomsVectorDeterminator,
     private val keyConverter: KeyConverter
 ) {
 
+    companion object {
+        const val MAX_DAYS_IN_PAST = 14
+        const val VECTOR_LENGTH = 15
```

Comment:
Ask: I find this variable/constant name name too unspecific. Of what vector is this the length? It might be worth moving this into the DaysSinceOnsetOfSymptomsVectorDeterminator directly; such a parameter does not exist for the TransmissionRiskVectorDeterminator.

Reply:
@ralfgehrer